pkg/model/awsmodel/firewall.go

```diff
@@ -192,22 +196,33 @@ func (b *FirewallModelBuilder) applyNodeToMasterBlockSpecificPorts(c *fi.Cloudup
 			}
 			for _, protocol := range protocols {
 				awsName := strconv.Itoa(int(protocol))
-				name := awsName
+
 				switch protocol {
 				case ProtocolIPIP:
-					name = "ipip"
+					awsName = "ipip"
```

Comment:
`ipip` is not a valid value for the `IpProtocol` field of an `IpPermission`.

> 	// The IP protocol name (tcp, udp, icmp, icmpv6) or number (see Protocol Numbers
	// (http://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml)).